platform-controller/src/test/java/org/hobbit/controller/docker/ContainerTerminationCallbackImplTest.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.hobbit.controller.docker;
 
+import org.hobbit.controller.containers.docker.ContainerTerminationCallbackImpl;
```

Comment:
When the tested class moves into a different package, the test classes should move, too.